<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tano</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../data/logo.qrc">
    <normaloff>:/logo/48x48/logo.png</normaloff>:/logo/48x48/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pageMain">
       <layout class="QVBoxLayout" name="verticalLayout_6"/>
      </widget>
      <widget class="QWidget" name="pagePlayback">
       <layout class="QVBoxLayout" name="verticalLayout_12">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="VlcVideoWidget" name="videoWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pageRecorder">
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="Recorder" name="recorder" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>650</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRecorder">
    <property name="title">
     <string>&amp;Recorder</string>
    </property>
    <addaction name="actionRecorder"/>
    <addaction name="actionTimers"/>
    <addaction name="separator"/>
    <addaction name="actionRecord"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>&amp;Options</string>
    </property>
    <addaction name="actionEditPlaylist"/>
    <addaction name="actionSettings"/>
    <addaction name="actionSettingsShortcuts"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionUpdate"/>
    <addaction name="actionSupport"/>
    <addaction name="actionDonate"/>
    <addaction name="actionAbout"/>
    <addaction name="actionAboutQt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>Vie&amp;w</string>
    </property>
    <addaction name="actionTop"/>
    <addaction name="actionLite"/>
    <addaction name="actionFullscreen"/>
    <addaction name="separator"/>
    <addaction name="actionControls"/>
    <addaction name="actionInfoPanel"/>
   </widget>
   <widget class="QMenu" name="menuAudio">
    <property name="title">
     <string>&amp;Audio</string>
    </property>
    <addaction name="actionMute"/>
    <addaction name="actionVolumeUp"/>
    <addaction name="actionVolumeDown"/>
   </widget>
   <widget class="QMenu" name="menuMedia">
    <property name="title">
     <string>&amp;Playback</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionStop"/>
    <addaction name="actionBack"/>
    <addaction name="actionNext"/>
    <addaction name="actionTeletext"/>
   </widget>
   <widget class="QMenu" name="menuVideo">
    <property name="title">
     <string>&amp;Video</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenUrl"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSchedule"/>
    <addaction name="separator"/>
    <addaction name="actionTray"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuMedia"/>
   <addaction name="menuAudio"/>
   <addaction name="menuVideo"/>
   <addaction name="menuRecorder"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenToolbar"/>
   <addaction name="separator"/>
   <addaction name="actionSchedule"/>
   <addaction name="separator"/>
   <addaction name="actionRecorder"/>
   <addaction name="actionEditPlaylist"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QDockWidget" name="dockControls">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable|QDockWidget::DockWidgetVerticalTitleBar</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Controls</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockControlsContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QVBoxLayout" name="osdLayout">
     <property name="margin">
      <number>0</number>
     </property>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBarRecorder">
   <property name="windowTitle">
    <string>Recorder Toolbar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRecordQuick"/>
   <addaction name="actionRecord"/>
   <addaction name="separator"/>
   <addaction name="actionRecordTimer"/>
  </widget>
  <widget class="QDockWidget" name="dockInfo">
   <property name="features">
    <set>QDockWidget::DockWidgetClosable</set>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockContents">
    <layout class="QVBoxLayout" name="dockLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="stackedWidgetDock">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="pagePlaylist">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="labelPlaylistIcon">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="pixmap">
              <pixmap resource="../../data/icons.qrc">:/icons/16x16/video.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelPlaylistName"/>
           </item>
           <item>
            <widget class="QPushButton" name="buttonPlaylistSearch">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>30</height>
              </size>
             </property>
             <property name="icon">
              <iconset resource="../../data/icons.qrc">
               <normaloff>:/icons/16x16/search.png</normaloff>:/icons/16x16/search.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonPlaylistClose">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="icon">
              <iconset resource="../../data/icons.qrc">
               <normaloff>:/icons/16x16/close.png</normaloff>:/icons/16x16/close.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="PlaylistDisplayWidget" name="playlistWidget" native="true"/>
         </item>
         <item>
          <widget class="QWidget" name="blank" native="true">
           <property name="maximumSize">
            <size>
             <width>1</width>
             <height>1</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSchedule">
           <property name="text">
            <string>Schedule</string>
           </property>
           <property name="icon">
            <iconset resource="../../data/icons.qrc">
             <normaloff>:/icons/16x16/calendar.png</normaloff>:/icons/16x16/calendar.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="pageSchedule">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="labelScheduleIcon">
             <property name="minimumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="pixmap">
              <pixmap resource="../../data/icons.qrc">:/icons/16x16/calendar.png</pixmap>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="labelScheduleName">
             <property name="text">
              <string>&lt;b&gt;Schedule&lt;/b&gt;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="buttonScheduleClose">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
             <property name="icon">
              <iconset resource="../../data/icons.qrc">
               <normaloff>:/icons/16x16/close.png</normaloff>:/icons/16x16/close.png</iconset>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="EpgScheduleChannel" name="scheduleWidget" native="true"/>
         </item>
         <item>
          <widget class="QPushButton" name="buttonScheduleBack">
           <property name="text">
            <string>Back</string>
           </property>
           <property name="icon">
            <iconset resource="../../data/icons.qrc">
             <normaloff>:/icons/16x16/go-previous.png</normaloff>:/icons/16x16/go-previous.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="../../data/logo.qrc">
     <normaloff>:/logo/48x48/logo.png</normaloff>:/logo/48x48/logo.png</iconset>
   </property>
   <property name="text">
    <string>About Tano</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/exit.png</normaloff>:/icons/24x24/exit.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/settings.png</normaloff>:/icons/24x24/settings.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/fullscreen.png</normaloff>:/icons/24x24/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Fullscreen</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/open.png</normaloff>:/icons/24x24/open.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open channel list</string>
   </property>
   <property name="toolTip">
    <string>Open channel list</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/open.png</normaloff>:/icons/24x24/open.png</iconset>
   </property>
   <property name="text">
    <string>Open &amp;File</string>
   </property>
  </action>
  <action name="actionEditPlaylist">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/edit.png</normaloff>:/icons/24x24/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit Channels</string>
   </property>
   <property name="toolTip">
    <string>Edit Channels</string>
   </property>
  </action>
  <action name="actionOpenUrl">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/open.png</normaloff>:/icons/24x24/open.png</iconset>
   </property>
   <property name="text">
    <string>Open URL</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/update.png</normaloff>:/icons/24x24/update.png</iconset>
   </property>
   <property name="text">
    <string>Check for updates</string>
   </property>
  </action>
  <action name="actionOpenToolbar">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/open.png</normaloff>:/icons/24x24/open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/media-playback-start.png</normaloff>:/icons/24x24/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/media-playback-stop.png</normaloff>:/icons/24x24/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionTop">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/top.png</normaloff>:/icons/24x24/top.png</iconset>
   </property>
   <property name="text">
    <string>Always on top</string>
   </property>
  </action>
  <action name="actionTray">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/interface.png</normaloff>:/icons/24x24/interface.png</iconset>
   </property>
   <property name="text">
    <string>Hide to tray</string>
   </property>
   <property name="toolTip">
    <string>Hide to tray</string>
   </property>
  </action>
  <action name="actionLite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/interface.png</normaloff>:/icons/24x24/interface.png</iconset>
   </property>
   <property name="text">
    <string>Simple Mode</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/media-skip-forward.png</normaloff>:/icons/24x24/media-skip-forward.png</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/media-skip-backward.png</normaloff>:/icons/24x24/media-skip-backward.png</iconset>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/volume-muted.png</normaloff>:/icons/24x24/volume-muted.png</iconset>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
  </action>
  <action name="actionVolumeUp">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/go-up.png</normaloff>:/icons/24x24/go-up.png</iconset>
   </property>
   <property name="text">
    <string>Volume up</string>
   </property>
  </action>
  <action name="actionVolumeDown">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/go-down.png</normaloff>:/icons/24x24/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Volume down</string>
   </property>
  </action>
  <action name="actionRecorder">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/recorder.png</normaloff>:/icons/24x24/recorder.png</iconset>
   </property>
   <property name="text">
    <string>Recorder</string>
   </property>
  </action>
  <action name="actionRecordQuick">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/media-record.png</normaloff>:/icons/24x24/media-record.png</iconset>
   </property>
   <property name="text">
    <string>Quick record</string>
   </property>
   <property name="toolTip">
    <string>Quick record</string>
   </property>
  </action>
  <action name="actionTimers">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/timer.png</normaloff>:/icons/24x24/timer.png</iconset>
   </property>
   <property name="text">
    <string>Timers</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/qt.png</normaloff>:/icons/24x24/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionRecord">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/media-playback-stop.png</normaloff>:/icons/24x24/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop recording</string>
   </property>
   <property name="toolTip">
    <string>Stop recording</string>
   </property>
  </action>
  <action name="actionControls">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/control.png</normaloff>:/icons/24x24/control.png</iconset>
   </property>
   <property name="text">
    <string>Controls panel</string>
   </property>
  </action>
  <action name="actionSchedule">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/calendar.png</normaloff>:/icons/24x24/calendar.png</iconset>
   </property>
   <property name="text">
    <string>Schedule</string>
   </property>
  </action>
  <action name="actionInfoPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/information.png</normaloff>:/icons/24x24/information.png</iconset>
   </property>
   <property name="text">
    <string>Info panel</string>
   </property>
  </action>
  <action name="actionPlaylistFullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/fullscreen.png</normaloff>:/icons/24x24/fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Enable Info panel in Fullscreen mode</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/donate.png</normaloff>:/icons/24x24/donate.png</iconset>
   </property>
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionTeletext">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/teletext.png</normaloff>:/icons/24x24/teletext.png</iconset>
   </property>
   <property name="text">
    <string>Teletext</string>
   </property>
  </action>
  <action name="actionSettingsShortcuts">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/shortcuts.png</normaloff>:/icons/24x24/shortcuts.png</iconset>
   </property>
   <property name="text">
    <string>Keyboard shortcuts</string>
   </property>
  </action>
  <action name="actionRecordNow">
   <property name="checkable">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionRecordTimer">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/add.png</normaloff>:/icons/24x24/add.png</iconset>
   </property>
   <property name="text">
    <string>Add timer</string>
   </property>
  </action>
  <action name="actionSupport">
   <property name="icon">
    <iconset resource="../../data/icons.qrc">
     <normaloff>:/icons/24x24/help.png</normaloff>:/icons/24x24/help.png</iconset>
   </property>
   <property name="text">
    <string>Support</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlaylistDisplayWidget</class>
   <extends>QWidget</extends>
   <header>ui/playlist/PlaylistDisplayWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>VlcVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">vlc-qt/VideoWidget.h</header>
   <container>1</container>
   <slots>
    <signal>fullscreen()</signal>
    <slot>toggleFullscreen()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Recorder</class>
   <extends>QWidget</extends>
   <header>ui/recorder/Recorder.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EpgScheduleChannel</class>
   <extends>QWidget</extends>
   <header>ui/epg/EpgScheduleChannel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../data/icons.qrc"/>
  <include location="../../data/logo.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionFullscreen</sender>
   <signal>triggered()</signal>
   <receiver>videoWidget</receiver>
   <slot>toggleFullscreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>318</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoWidget</sender>
   <signal>fullscreen()</signal>
   <receiver>actionFullscreen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>318</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
